WordPress/Sniffs/NamingConventions/ValidVariableNameSniff.php

```diff
@@ -236,12 +246,30 @@ public static function isSnakeCase( $var_name ) {
 	 * Merge a custom whitelist provided via a custom ruleset with the predefined whitelist,
 	 * if we haven't already.
 	 *
+	 * @param PHP_CodeSniffer_File $phpcs_file The file being scanned.
+	 *
 	 * @return void
 	 */
-	protected function mergeWhiteList() {
-		if ( false === $this->addedCustomVariables && ! empty( $this->customVariablesWhitelist ) ) {
-			$this->whitelisted_mixed_case_member_var_names = array_merge( $this->whitelisted_mixed_case_member_var_names, $this->customVariablesWhitelist );
-			$this->addedCustomVariables = true;
+	protected function mergeWhiteList( $phpcs_file ) {
+		if ( ! empty( $this->customVariablesWhitelist ) ) {
```

Comment:
Ahh, good catch, I intended that but forgot to double check it in the end result.